internal/controllers/synthesis/slicecleanup.go

```diff
@@ -39,36 +43,20 @@ func (c *sliceCleanupController) Reconcile(ctx context.Context, req ctrl.Request
 	}
 
 	owner := metav1.GetControllerOf(slice)
-
-	// We only get the composition if it exists
-	// It shouldn't be possible that it doesn't exist, but still worth handling in case anyone creates an ad-hoc resource slice for some reason (it won't do anything tho)
```

Comment:
Apparently it was very possible 😭

Reply:
I should tally up all of my comments like this that ended up being removed in bugfix PRs 😆